### PR TITLE
Change spacing between logo circle and text

### LINKED ISSUE #440: Fix the logo spacing
There is a larger gap between the logo and the text in the top right to accommodate the collapsible sidebar. To remedy this we could animate the gap closer when the sidebar is open.

Re: https://github.com/canonical-web-and-design/jaas-dashboard/pull/431#issuecomment-616412388

### PR DESCRIPTION
**Pull request by hatched**
## Done

Animate the Juju and JAAS logo text so that it maintains proper spacing when the sidebar is open, but provides appropriate spacing to avoid artifacts when the sidebar is closed.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- When the app loads the logo in the top left corner should have the appropriate spacing between the logo circle and text.
- Click to view a model details and the sidebar should animate closed and you should not see any text artifacts by the logo circle.
- Hover to expand the sidebar and the text should animate closer.
- Collapse the sidebar and the text should animate away.

## Details

Fixes #440 

## Screenshots
Screen Shot 2020-04-20 at 3 53 49 PM
Screen Shot 2020-04-20 at 3 54 02 PM

### CODE REVIEW

File: src/components/PrimaryNav/_primary-nav.scss [hunk 1]
```diff
@@ -113,6 +113,11 @@
 
     @at-root {
       .is-collapsed {
+        .p-primary-nav__logo-text {
+          @include vf-animation(padding, slow, ease-in-out);
+          padding: 0 0 0 0.6rem;
```
Comment (anthonydillon) [inline src/components/PrimaryNav/_primary-nav.scss:118]: can you just use `padding-left`?

**Comment by hatched** [comment]
Good call, the opacity looks better, I've updated it to use opacity.

**Comment by barrymcgee** [comment]
@spencerbygraves @hatched GIF of logo change

Screen Recording 2020-04-21 at 09 53 pm

**Comment by spencerbygraves** [comment]
@barrymcgee thanks for the GIF.

It's better but still not quite right. Below you can see that the gap is 5px at that size:

image

**Comment by spencerbygraves** [comment]
Also meant to say that "Juju" is too big relative to the circle:

image

**Comment by hatched** [comment]
The spacing requirements weren't outlined in the original issue and the provided assets do not allow for these changes. @spencerbygraves will provide new svg assets for both the "Juju" and "JAAS" text logos. 👍

**Comment by spencerbygraves** [comment]
@hatched there doesn't appear to be any extra white space in the asset:

image

https://assets.ubuntu.com/v1/a85f7947-juju_black-text-only.svg

**Comment by hatched** [comment]
I just manually edited the SVG until I got something that was what I think you wanted. Both the Juju and JAAS logos are 25px high and have a 5px gap from the logo.

**Pull request merged**